app/models/user.rb

```diff
@@ -365,7 +365,8 @@ def identity_verified?(service_provider: nil)
   end
 
   def identity_verified_with_selfie?
-    active_profile&.idv_level == 'unsupervised_with_selfie'
+    biometric_comparison_idv_levels = ['unsupervised_with_selfie', 'in_person']
```

Comment:
should we make this into a constant? also personally I like the `%w[....]` syntax for these kinds of arrays

Reply:
Yeah, let constantify this.